core/test/unit/web/middleware/uncapitalise_spec.js

```diff
@@ -123,24 +123,21 @@ describe('Middleware: uncapitalise', function () {
             done();
         });
     });
-
+    
     describe('An API request', function () {
-        let apiVersion;
-        describe(`for ${apiVersion}`, function () {
-            beforeEach(function () {
-                apiVersion = ['v0.1', 'v2', 'canary', 'v10'];
-            });
-            it('does nothing if there are no capitals', function (done) {
-                apiVersion.forEach((apiVersion) => {
+    // NOTE: there is no good way to substitute dynamic test setup based on an array of values, see {https://github.com/TryGhost/Ghost/pull/11154}     
+    // eslint-disable-next-line  ghost/mocha/no-setup-in-describe  
```

Comment:
@dhuang612 awesome :smile: I think, this is exactly how we should deal with these types of test case iterations :+1: 

Think all is left in this PR is to apply (or rather revert) changes to other places where we iterate over multiple test cases and leave a similar comment there. Think we are very close to merging this gigantic change!

Reply:
Hey @gargol sorry I've been really sick and resting. Just starting to get back into finishing up this pr. 

I was wondering, when you talk about the other test case iterations that need to be reverted back and commented. Are you talking about these?
**pages_spec.js**
```
describe('field formats', function () {
            const fieldMap = {
                title: [123, new Date(), _.repeat('a', 2001)],
                slug: [123, new Date(), _.repeat('a', 192)],
                mobiledoc: [123, new Date()],
                feature_image: [123, new Date(), 'random words'],
                featured: [123, new Date(), 'abc'],
                status: [123, new Date(), 'abc'],

```